mubench.reviewsite/tests/StoreReviewTest.php

```diff
@@ -49,7 +49,7 @@ function setUp()
             "result" => "success",
             "runtime" => "42.1",
             "number_of_findings" => "23",
-            "detector" => $this->detector->id,
+            "-custom-stat-" => "-stat-val-",
```

Comment:
Why do you have to touch this test? Storing reviews has nothing to do with storing run info, has it?